allauth/socialaccount/providers/metamask/templates/metamask/auth.html

```diff
@@ -0,0 +1,73 @@
+{% load static %}
+<div id="personalSign">  </div>
+<HR>
+<div id="personalSignResult">  </div>
+<HR>
```

Comment:
Are these HR's something that is visible and needs to be styled/tweaked when actually using this provider in a real project?  Can't we drop these altogether and use CSS to style the div ?

Reply:
I'm dropping these divs and HRs, they were for debug mostly.